cdist/conf/type/__docker/manifest

```diff
@@ -24,7 +24,7 @@ state=$(cat "$__object/parameter/state")
 
 case "$os" in
     centos)
-        if (source "$__global/explorer/os_release" && [ "${VERSION_ID}" = "7" ]); then
+        if (. "$__global/explorer/os_release" && [ "${VERSION_ID}" = "7" ]); then
```

Comment:
$__global/explorer/os_release is supposed to executed, not sourced.

Reply:
@asteven Aren't **source** and **.**(dot) the same and only dot is defined by POSIX?

Reply:
They are. But explorers are **executables**. There is no need to source them at all.

Reply:
@asteven  As I understand in the above code "$__global/explorer/os_release" is the output of os_release explorer after it has been run on target host. This output is read and executed using *dot* in the current env and then it is checked if VERSION_ID equals 7. I understand that the output of os_release contains such shell compatible variable definition.

Reply:
@asteven Ping